scipy/stats/tests/test_multicomp.py

```diff
@@ -0,0 +1,103 @@
+import numpy as np
+import pytest
+from numpy.testing import assert_allclose
+
+from scipy import stats
+from scipy.stats._multicomp import pvalue_dunnett, DunnettResult
+
+
+class TestDunnett:
```

Comment:
Please add IV tests

Reply:
Let me know if what I have added is enough.